crates/rooch-indexer/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=/Users/Baichuan/.rooch/local/roochdb/indexer.sqlite
```

Comment:
Is the `DATABASE_URL` must an absolute path?

Reply:
the `.env` is only for dev, and It should be removed from the crate.